src/Step/WpConfigStep.php

```diff
@@ -100,17 +86,18 @@ public function run(Config $config, Paths $paths): int
 
         $autoload = $paths->vendor('autoload.php');
 
+        /** @var bool $cacheEnv */
         $cacheEnv = $config[Config::CACHE_ENV]->unwrapOrFallback(true);
 
         /** @var string $earlyHookFile */
         $earlyHookFile = $config[Config::EARLY_HOOKS_FILE]->unwrapOrFallback('');
-        if ($earlyHookFile) {
+        if ($earlyHookFile !== '') {
             $earlyHookFile = $this->relPath("{$from}/index.php", $earlyHookFile, false);
         }
 
         /** @var string $envBootstrapDir */
         $envBootstrapDir = $config[Config::ENV_BOOTSTRAP_DIR]->unwrapOrFallback('');
-        if ($envBootstrapDir) {
+        if ($envBootstrapDir === '') {
             $envBootstrapDir = $this->relPath($from, $paths->root($envBootstrapDir));
         }
```

Comment:
I think it's the same as for `$earlyHookFile`, we need to check `$envBootstrapDir !== ''`

Reply:
Thank you very much. Fixed!